lib/active_record/connection_adapters/mysql2rgeo_adapter.rb

```diff
@@ -137,6 +137,10 @@ def quote(value)
         end
       end
 
+      def with_connection
+        yield self
+      end
```

Comment:
Required by this change to ActiveRecord: https://github.com/rails/rails/pull/51162/files#diff-22750093c2231cef8c58e7e94a0a9266267d33bd990c607475e55fe285fccb8aR45

I only saw this `with_connection` method getting called by tests in this gem that were dumping the schema. I'm sure something more sophisticated could be created for this, however it fixed the failing tests.